spec/support/features/idv_helper.rb

```diff
@@ -68,11 +68,19 @@ def click_idv_continue
   end
 
   def click_idv_address_choose_phone
-    click_link t('idv.buttons.activate_by_phone')
+    # we're capturing the click on the label element via the unique "for" attribute
```

Comment:
same comment on comments 🙂

Reply:
I think we should keep it here, because the click handler is a bit unique. There are multiple elements within the label, and clicking the label itself is the only thing that matches how the CSS is actually setting the hidden input to true. Since it's both not immediately clear and a relatively unfamiliar pattern, I think it's worth commenting.

Reply:
Having a slack conversation about giving the user an error if nothing is chosen: https://gsa-tts.slack.com/archives/C0NGESUN5/p1511195802000213

Reply:
@tbaxter-18f and I just chatted offline and decided that verification via text should be pre-selected as the vast majority of folks will want to choose that. That eliminates the need to provide an error if nothing is selected.